docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,173 @@
+
+The former option has the advantage of persisting your changes whenever you bring down and bring back up the node. The downside is that you will have to port the options you customized to the new YAML when upgrading to a new binary version (you should use the latest YAML when upgrading to a new version). By using a file like `./env<name of node>.list` that contains your environment variables, it is easy to port your changes to a new YAML.
+
+Either way, here are some values in `core.yaml` you might want to edit.
+
+* `localMspID`: this is the name of the local MSP, generated by your CA, of your peer organization. This is where your peer organization admins will be listed.
+
+* `mspConfigPath`: the place where the local MSP for the peer is located. Note that it is a best practice to mount this volume external to your container.
+
+* `peer/address`: represents the endpoint to other peers in the same organization, an important consideration when establishing gossip communication within an organization.
+
+* `# TLS enabled`, `Enabled: true`: This is where you specify whether you want to [enable TLS](enable_tls.html). If you set this value to `true`, you will have to specify the locations of the relevant TLS certificates. Note that all of the nodes in a network (both the peers and the ordering nodes, must either all have TLS enabled or not enable. For production networks, it is highly recommended to enable TLS). As with your MSP, it is a best practice to mount this volume external to your container.
+
+* `ledger`: users have a number of decisions to make about their ledger, including the state database type (level or CouchDB), and its location (specified in `fileSystemPath` --- not that for CouchDB in particular that it is a best practice to operate your state database external to the peer).
+
+* `gossip`: there are a number of configuration options to think about when setting up [gossip](gossip.html), including the `externalEndpoint` (which makes peers discoverable to peers owned by other organizations) as well as the `bootstrap` address (which identifies a peer in the peer's own organization).
+
+* `externalBuilders`: this field is important to set when using [Chaincode as an external service](cc_service.html).
+
+When you're comfortable with how your peer has been configured, how your volumes are mounted, and your backend configuration, you can run the command to launch the peer.
+
+### Create an ordering node
+
+Unlike the creation of a peer, you will need to create a genesis block (or reference a block that has already been created, if adding an ordering node to an existing ordering service) and specify the path to it before launching the ordering node.
+
+In Fabric, this configuration file for ordering nodes is known as `orderer.yaml`. You can find a sample `orderer.yaml` configuration file [in the sampleconfig directory of Hyperledger Fabric](https://github.com/hyperledger/fabric/blob/master/sampleconfig/orderer.yaml).
```

Comment:
Does this imply that the orderer.yaml is the genesis block?

Reply:
Clarified this.